Mlem/App/Views/Pages/Instance/InstanceUptimeView.swift

```diff
@@ -0,0 +1,280 @@
+//
+//  InstanceUptimeView.swift
+//  Mlem
+//
+//  Created by Sjmarf on 28/01/2024.
+//
+
+import Charts
+import MlemMiddleware
+import SwiftUI
+
+struct InstanceUptimeView: View {
+    @Environment(\.accessibilityDifferentiateWithoutColor) var diffWithoutColor: Bool
+    @Environment(Palette.self) var palette
+    
+    @State var showingExactTime: Bool = false
+    @State var showingAllDowntimes: Bool = false
+    
+    let instance: any Instance
+    let uptimeData: UptimeData
+    
+    @ViewBuilder
+    var body: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            section { summary }
+                .padding(.bottom, 10)
+            section("Recent Checks") {
+                recentChecks
+                    .padding(.horizontal)
+                    .padding(.vertical, 15)
+            }
+            .padding(.top, 20)
+            // String interpolation used here to avoid localizing the number
+            footnote("Mlem will refresh this automatically every \(30) seconds.")
```

Comment:
*nit*

Wording can be more concise (maybe even omit "automatically"?)

```suggestion
            footnote("Automatically refreshes every \(30) seconds.")
```